24552b8fd1291c9c8046d14dd813b9a4f22bf883 prevent harmless LVM warnings from grub-probe
 grub-core/osdep/unix/getroot.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grub-core/osdep/unix/getroot.c b/grub-core/osdep/unix/getroot.c
index 74f69116d..8e143beb2 100644
--- a/grub-core/osdep/unix/getroot.c
+++ b/grub-core/osdep/unix/getroot.c
@@ -609,7 +609,7 @@ grub_util_pull_lvm_by_command (const char *os_dev)
   /* by default PV name is left aligned in 10 character field, meaning that
      we do not know where name ends. Using dummy --separator disables
      alignment. We have a single field, so separator itself is not output */
-  argv[0] = "vgs";
+  argv[0] = "LVM_SUPPRESS_FD_WARNINGS=1 vgs";
   argv[1] = "--options";
   if (vgid)
     argv[2] = "vg_uuid,pv_name";
-- 
2.39.2

